scripts/common.py

```diff
@@ -215,9 +215,12 @@
     'logged-in-user-tests/click-all-buttons-in-thanks-for-donating-page' +
     '.spec.js',
     'practice-question-admin-tests/add-and-remove-contribution-rights.spec.js',
+    'practice-question-submitter-tests' +
+    '/suggest-medium-drag-and-drop-sort-question.spec.js',
+    'practice-question-submitter-tests/suggest-hard-fraction-input-question' +
+    '.spec.js',
```

Comment:
I don't see this spec file in the PR. Could you double-check this?

Reply:
I ended up not including the question submitter spec files for this PR. So I believe both spec filenames should not be in this file.